deny.toml

```diff
@@ -99,6 +99,10 @@ name = "async-executor"
 [[bans.deny]]
 name = "smol"
 
+[[bans.deny]]
+# We want to use rustls instead of native-tls.
+name = "postgres-native-tls"
```

Comment:
can we ban `native-tls` directly, rather than `postgres-native-tls`?

Reply:
yesterday I've tried to figure out for 2-3 hours which dependency was enabling the native-tls backend of reqwest. After a lot of trial and error, it turned out to be the azure SDK. Apparently if you turn off default features, it works.

I've originally also suspected opentelemetry but now that the patch is simpler I think it can be added to the PR.

So now I'm banning `native-tls` and `openssl` instead of `postgres-native-tls`.